config/boards/README.md

```diff
@@ -4,6 +4,9 @@
 | BOARD_NAME | welcome text and hostname |
 | BOARDFAMILY | [sun8i, sun7i, rockchip64](../sources) |
 | BOOTCONFIG | name of u-boot config |
+| BOOTCONFIG_DEFAULT | name of u-boot config for default branch |
```

Comment:
https://github.com/armbian/build/blob/master/lib/configuration.sh#L127

Its related to branch namings: LEGACY CURRENT DEV (default and next are deprecated)